MoltenVK/MoltenVK/GPUObjects/MVKImage.mm

```diff
@@ -425,7 +429,7 @@
             _mtlTexelBufferOffset = getDeviceMemoryOffset();
         } else {
             // Create our own buffer for this.
```

Comment:
Technically, the Vulkan spec mandates a memory-binding activity should only occur once for an image, meaning this function should only ever be called once by the app. But just to make the leak protection a little bit more defensive in case an app makes a mistake, it might be a good to include the following line in between this comment and  the `if()` statement:

```
if (_ownsTexelBuffer) { [_mtlTexelBuffer release]; }
```